src/sandbox/components/Grid.svelte

```diff
@@ -32,6 +35,8 @@
   let highlight = selection;
   let arrowRow, arrowCol;
 
+  $: if (selection && !document.querySelector("#ellx-node-navigator").classList.contains("hidden")) closeEditor();
```

Comment:
better to make NodeNavigator hidden flag part of the store. Let's discuss: this is a very good example when state management becomes necessary